tests/BenchmarkDotNet.Tests/Exporters/CommonExporterApprovalTests.cs

```diff
@@ -60,7 +60,7 @@ public void Exporters(string cultureInfoName)
                 exporter.ExportToLog(MockFactory.CreateSummary(config.WithCultureInfo(cultureInfo)), logger);
             }
 
-            Approvals.Verify(logger.GetLog());
+            Approvals.Verify(logger.GetLog(), x => x.Replace("\r", string.Empty));
```

Comment:
When I first ran the tests after the change I made, they were failing due to `\r\n` vs. `\n` line ending differences rather than the additional members for the memory stats (my laptop runs Windows).